src/components/common/CompoundLayout/components/Section.tsx

```diff
@@ -0,0 +1,28 @@
+import { Container, VStack } from "@chakra-ui/react"
+import { ReactNode } from "react"
+import { callCssVariable } from "../callCssVariable"
+import { LAYOUT_MAX_WIDTH_CSS_VAR } from "../constants"
+
+interface SectionProps {
+  children: ReactNode
+}
+
+export function MainSection({ children }: SectionProps) {
```

Comment:
Let's make 2 different files: `MainSection` & `HeaderSection`. We already have a `Section` component, so it'd be a bit confusing to have 2 components with the same name